bundle.md

```diff
@@ -1,39 +1,27 @@
-# Bundle
+# Filesystem Bundle
 
 ## Container Format
 
-This section defines a format for encoding a container as a *bundle* - a directory organized in a certain way, and containing all the necessary data and metadata for any compliant runtime to perform all standard operations against it.
+This section defines a format for encoding a container as a *filesystem bundle* - a set of files organized in a certain way, and containing all the necessary data and metadata for any compliant runtime to perform all standard operations against it.
 See also [OS X application bundles](http://en.wikipedia.org/wiki/Bundle_%28OS_X%29) for a similar use of the term *bundle*.
 
-The format does not define distribution.
-In other words, it only specifies how a container must be stored on a local filesystem, for consumption by a runtime.
-It does not specify how to transfer a container between computers, how to discover containers, or assign names or versions to them.
-Any distribution method capable of preserving the original layout of a container, as specified here, is considered compliant.
+The definition of a bundle is only concerned with how a container, and its configuration data, are stored on a local file system so that it can be consumed by a compliant runtime.
 
-A standard container bundle is made of the following 3 parts:
+A Standard Container bundle contains all the information needed to load and run a container.
+This includes the following three artifacts which MUST all reside in the same directory on the local filesystem:
 
-- A top-level directory holding everything else
-- One or more content directories
-- A configuration file
+1. `config.json` : immutable, host independent configuration.
+This file, which MUST be named `config.json`, contains settings that are host independent and application specific such as security permissions, environment variables and arguments.
```

Comment:
On Mon, Oct 05, 2015 at 03:40:11PM -0700, Michael Crosby wrote:
> > +This file, which MUST be named `config.json`, contains settings
> > that are host independent and application specific such as
> > security permissions, environment variables and arguments.
> 
> it's just an example, being vague is ok. 

I think it's misleading, because almost all of the security stuff is
going to be handled in runtime.json.  If we mean Linux capabilities,
it seems easy enough to just say that.

But whatever, there's a link to the config specs right after each
bullet point, so folks can see what's actually in the files ;).

@duglin, ‘git log -p -G 'security permissions'’ points to 7232e4b1
(specs: introduce the concept of a runtime.json, 2015-07-30, #88).
The only inline comment on anything in its general viscinity was the
conversation I had with @philips [1] that lead to me filing #107 with
a definition for “application” (since removed from that PR [2]).

[1]: https://github.com/opencontainers/specs/pull/88#discussion_r35930309
[2]: https://github.com/opencontainers/specs/pull/107#issuecomment-145619316